procedures/CodeBrowser.ipf

```diff
@@ -808,17 +843,20 @@ static Function saveLoad(procedure)
 	endif
 
 	numResults = Dimsize(SaveWavesWave[procedure.row][0], 0)
-	Redimension/N=(numResults, -1) declWave, lineWave
+	Redimension/N=(numResults, -1) declWave, lineWave, helpWave
 	if(numResults == 0)
 		debugPrint("no elements in save state")
 		return 0
 	endif
 
 	WAVE/T load0 = SaveWavesWave[procedure.row][0]
 	WAVE/I load1 = SaveWavesWave[procedure.row][1]
+	WAVE/T load2 = SaveWavesWave[procedure.row][2]
 	declWave[][0] = load0[p][0]
 	declWave[][1] = load0[p][1]
 	lineWave[] = load1[p]
+	helpWave[][0] = load2[p][0]
+	helpWave[][1] = load2[p][1] //@todo why not use q?
```

Comment:
Using `q` sounds like a good idea or?

Reply:
I am sure there was a reason for this work-around. I adopted the style from declWave, 2 lines up